### DISCUSSION TITLE
GitHub student pack

### DISCUSSION THREAD

**Discussion by HassanWebdev** [community/community#132471] [GitHub Education]
### Hi everyone,

Is there any technical issue so I didn't receive my GitHub benefits.it has been 3 days . If anyone know the issue is please let me know. Thanks

**Comment by NugrohoRahmanto** [comment]
just wait it i also get it after 7 days, and if you still not get it try to contact github admin

**Comment by adarsh-site** [comment]
generally it doesn't take this much amount of time. Just wait for couple of days and if it will not come up till then, try contacting support.

**Comment by adarsh-site** [comment]
hey, I just found this. maybe thats the reason.

"The GitHub team will have reduced availability between July 13th and July 21st. You may experience a slight delay in our support responses during this period."

**Comment by davevad93** [comment]
Hi @HassanWebdev ,

It may take up to a couple of weeks for the benefits to become available, it usually takes just a few days but the waiting time may vary depending on different factors such as the volume of applications.

When the benefits are available the status label will turn purple and change to:

`Congrats! Your request for @HassanWebdev was approved and your benefits are now available.`
